### PR TITLE
flexibee: 2019.3.0.3 -> 2019.3.0.7 and adding the server binary

### PR DESCRIPTION
**Pull request by mmahut**
###### Motivation for this change

ChangeLog https://www.flexibee.eu/k/prehled-verzi-flexibee/prehled-verzi-flexibee-2019/

And packaging the flexibee-server binary.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/office/flexibee/default.nix [hunk 1]
```diff
@@ -11,16 +11,25 @@ stdenv.mkDerivation rec {
...
+      --replace "/var/run" "/run"
+  '';
+
+
   installPhase = ''
     runHook preInstall
     cp -R usr/share/flexibee/ $out/
     install -Dm755 usr/bin/flexibee $out/bin/flexibee
+    install -Dm755 usr/sbin/flexibee-server $out/bin/flexibee-server
     wrapProgram  $out/bin/flexibee --set JAVA_HOME "${jre}"
```
Comment (prusnak) [inline pkgs/applications/office/flexibee/default.nix]: extra space after `wrapProgram`

**Comment by prusnak** [comment]
Running `flexibee-server` ends up with error:

```
$ flexibee-server 
mkfifo: cannot create fifo '/run/flexibee-17229.pipe': Permission denied
/nix/store/wfcdnsbxzfs81ir53gvq5jb909xz1pla-flexibee-2019.3.0.7/bin/.flexibee-server-wrapped: line 274: /var/log/flexibee.log: Permission denied
cat: /run/flexibee-17229.pipe: No such file or directory
```

**Comment by mmahut** [comment]
> Running `flexibee-server` ends up with error:

Yes, this will be run as a service under a specific user and systemd is needed to create these temporary files. It is a really nasty piece of software sadly.

Service module is coming :)

File: pkgs/applications/office/flexibee/default.nix (hunk 1 above)
Comment (mmahut) [reply pkgs/applications/office/flexibee/default.nix]: Thanks!

**Pull request merged**